src/store/modules/search.store.js

```diff
@@ -131,22 +140,21 @@ const actions = {
                     })
                 }
                 dispatch('setPinnedLocation', {
-                    coordinates: entry.coordinates,
+                    coordinates: entry.coordinate,
                     dispatcher: dipsatcherSelectResultEntry,
```

Comment:
In the old mapviewer, getting a feature through the search would select the feature and show its feature Info box. Here we simply zoom to it and add a pin on the map.
Do we want to keep the old mapviewer behavior, and if so, do we plan on doing this at a later date ?

Reply:
That might be a good idea to use `setSelectedFeatures`... I'll look into it